test/e2e/config/ironic-vbmc.yaml

```diff
@@ -0,0 +1,73 @@
+images:
+# Use locally built e2e images
+- name: quay.io/metal3-io/baremetal-operator:e2e
+  loadBehavior: tryLoad
+- name: quay.io/metal3-io/ironic:e2e
+  loadBehavior: tryLoad
+# Save some time and network by using cached images if available
+- name: quay.io/metal3-io/baremetal-operator:release-0.4
```

Comment:
Why `release-0.4`, it doesn't even have the BMO e2e?

Reply:
It's for upgrade test. We will install this version, then upgrade it to main.